### DISCUSSION TITLE
Power button?

### DISCUSSION THREAD

**Discussion by zheroz00** [riffnshred/nhl-led-scoreboard#358] [Dev and Ideas]
Anyone know how to incorporate a power button to toggle the power on and off?

**Comment by pjockey** [comment]
You can look at this for a start https://howchoo.com/g/mwnlytk3zmm/how-to-add-a-power-button-to-your-raspberry-pi

**Reply by zheroz00** [reply]
That should work. TY.

**Comment by riffnshred** [comment]
check out our IO section. https://github.com/riffnshred/nhl-led-scoreboard/blob/master/src/sbio/SBIO.md#sbio-pushbutton

While using other methods work, our system show on screen what the scoreboard is doing (rebooting, shutting down)

**Reply by pjockey** [reply]
I knew you had a way to do the power button, but do you think I could locate it? 

Thanks

**Reply by zheroz00** [reply]
Perfect. Thank you.

**Reply by riffnshred** [reply]
> I knew you had a way to do the power button, but do you think I could locate it?
> 
> Thanks

you mean in the config? or on Github?

**Reply by pjockey** [reply]
I thought i saw it in your wiki on GitHub.
